flowpm/raytracing.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Feb  3 12:29:19 2021
+
+@author: Denise Lanzieri
+"""
+import numpy as np
+import tensorflow as tf
+import tensorflow_addons as tfa
+import flowpm
+import flowpm.constants as constants
+
+
+def generate_matrix():
+  """ Generate the possible rotation matrices
```

Comment:
Add docstring docucmentation using the formating convention we adopted. See here https://www.tensorflow.org/community/contribute/code_style#tensorflow_conventions_and_special_uses

Reply:
done